File: sprinterBoardDAL/Models/Task.cs
namespace spriterBoardDAL.Models
{
    public class Task
    {
        public long Id{get;set;}
        public string Name{get;set;}
        public string Description{get;set;}
        public TaskStatus Status {get;set;}
        
        public virtual AppUser User { get; set; }
        public int Order{get;set;}
        
    }
}

File: sprinterBoardSPA/clientApp/src/app/model/task.ts
export class Task {
  constructor(id: any = null, name: string = "", description: string = "") {
    this.Id = id;
    this.Name = name;
    this.Description = description;
  }
  Id: any;
  Name: string;
  Description: string;
  UserId: string;
}
